### DISCUSSION TITLE
v1.7.5.1

### DISCUSSION THREAD

**Discussion by aluminum-ice** [aluminum-ice/pwnagotchi#26] [Announcements]
I got my hands on an RPi2ZW that uses the legacy 43430/1 chipset. With a small tweak, I managed to get my fork of pwnagotchi working on the legacy 43430/1 as well as the newer 43430/2 chipset. And I merged a PR to implement thread pooling to handle another source of crashes I've observed.

- c28188d497a748c6337c8f8bd6e20970ee07cb9d Thread Pooling for Plugin Event Callbacks in Pwnagotchi (Thank you LegendEvent for the PR)
- 107af9a734e5eab8efee3ba92e128c7a6d15105a Ensure wlan0 is brought down after mon0 is brought up to prevent crashes with the RPi2ZW that use 43430/1

<hr /><em>This discussion was created from the release <a href='https://github.com/aluminum-ice/pwnagotchi/releases/tag/v1.7.5.1'>v1.7.5.1</a>.</em>

**Comment by TenThousand1** [comment]
Flashing this now, will update in approx. one hour.

**Comment by naturekid72** [comment]
For me it is working fine. Everything works. A bunch of plugins running as expected. Thanks for all the good work!!
